resources/tests.mjs

```diff
@@ -18,8 +18,8 @@ Suites.enable = function (names) {
 };
 
 Suites.push({
-    name: "VanillaJS-TodoMVC",
-    url: "todomvc/vanilla-examples/vanillajs/index.html",
+    name: "TodoMVC-JavaScript-ES5",
```

Comment:
I like the suggestion since it will allow us to namespace ScenarioName-WorkloadName when looking at results and the codebase. Though this will likely require some kind of migration for any tooling that's tracking subtests so let's make sure everyone is OK with that (cc @rniwa, and I'll confirm with our team)

Reply:
Confirmed this would be fine for us to change

Reply:
This seems fine to me.